doc/source/basics.rst

```diff
@@ -10,6 +10,14 @@
    np.set_printoptions(precision=4, suppress=True)
    from pandas.compat import lrange
    options.display.max_rows=15
+   import matplotlib.pyplot as plt
+   plt.close('all')
+   import matplotlib
+   try:
+      matplotlib.style.use('ggplot')
+   except AttributeError:
+      options.display.mpl_style = 'default'
+
```

Comment:
Are these import still needed? As you don't seem to use a plotting example anymore now?